src/twitter.js

```diff
@@ -39,7 +39,7 @@ class Twitter {
 
     const json = await fetch(url.toString(), {
       headers: {
-        Authorization: await this.credentials.authorizationHeader(url),
+        Authorization: await this.credentials.authorizationHeader(url, 'GET'),
```

Comment:
shouldn't this be `authorizationHeader(url, { method: 'GET' })`?

*likewise with the other call-sites*

Reply:
According to its document, the default is 'GET', so it's optional.
https://github.com/ddo/oauth-1.0a#authorize_-options-_